src/components/ProductListPage/CheckBoxList/CheckBox/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './CheckBox';
```

Comment:
I this component is going to be used outside of CheckboxList ?

Reply:
it may be used, I will move it to common folder